<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alret</class>
 <widget class="QMainWindow" name="alret">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CloseBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ooooooo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancleBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>390</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>2222</string>
    </property>
   </widget>
   <widget class="QLabel" name="TitleLab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="InerLab">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
